mpd_oled v1.0.7

supercrab opened the pull request:
Node_modules deleted and package.json updated

balbuze commented:
Hi!
I managed to include a compiled version of CAVA! Much more lighter and full requirement for Volumio!
Feel free to reuse and update your package ;-)
[mpd_oled.zip](https://github.com/volumio/volumio-plugins-sources/files/9275115/mpd_oled.zip)

supercrab commented:
> Hi!
> I managed to include a compiled version of CAVA! Much more lighter and full requirement for Volumio!
> Feel free to reuse and update your package ;-)
> [mpd_oled.zip](https://github.com/volumio/volumio-plugins-sources/files/9275115/mpd_oled.zip)
> edit : remove or update screenshots or change format to reduce the size

Wow that’s amazing!  I’ll have a look and include in a new PR :)  what files did you change? @balbuze

supercrab commented:
Will reduce the file size of the images somehow. What size should the plugin be?

balbuze commented:
I changed

-Install.sh
-Package.json

And added in the plugin folder 
-cava
-cava.psf (or something like that, I can't check now)

Don't forget to remove node_modules in your next PR!

supercrab commented:
> I changed
> 
> -Install.sh -Package.json
> 
> And added in the plugin folder -cava -cava.psf (or something like that, I can't check now)
> 
> Don't forget to remove node_modules in your next PR!
> 
> For the size, we are far from the plugin limit 12Mo but it is IMHO useless to have 1.6Mo for a simple outdated screenshot;-)

Cool thanks!

I have removed my node_plugins folder from my fork. Can you confirm?  It shouldn’t be in this PR and the one before it  thanks.


I can update the screenshots too. Should only need to be a few 100k!

balbuze commented:
There is still two issues for me:
- on my rpi3, it does not start properly when it boots. The oled is on, with a fix time displayed and nothing happens when play start. I have to go in plugin settings and press save to make it work.
- the time displayed is two hours later from the real time. I'm in France with summer time, two hours later. What option to fix that?

supercrab commented:
> There is still two issues for me:
> 
> * on my rpi3, it does not start properly when it boots. The oled is on, with a fix time displayed and nothing happens when play start. I have to go in plugin settings and press save to make it work.
> * the time displayed is two hours later from the real time. I'm in France with summer time, two hours later. What option to fix that?

@balbuze 

- I was getting an intermittent issue with no visualiser working when  I ran `volumio plugin install && volumio vrestart`.  I've ran just `volumio plugin install` to install the plugin 4 times and the issue seems to be fixed now.
- You must set the timezone using this command `sudo dpkg-reconfigure tzdata`  I'm not sure if this can be automated though!
